### ISSUE TITLE
Extra space beneath "What's Happening" feed, especially noticeable on iPad + Safari

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1520]
There is some extra space underneath the last activity in the What's Happening feed, but it is especially noticeable if you are using an iPad (Safari). This exists on both Staging & Production.

**Chrome 60, Mac OS El Capitan**
<img width="998" alt="screen shot 2017-09-08 at 11 07 04 am" src="https://user-images.githubusercontent.com/1320456/30218034-e7e7dec6-9485-11e7-9caf-29494b20a895.png">

**Small & Large iPads, Safari**
- Small apple tablet, iPad 10.3.1, no. 3 (in cart), Safari
- Large apple tablet, iPad 9.3.5, no. 10 (in cart), "cold pizza", Safari
![img_9121](https://user-images.githubusercontent.com/1320456/30218049-f6b2dfa0-9485-11e7-8c10-c98a4da62cc9.JPG)
![img_9120](https://user-images.githubusercontent.com/1320456/30218050-f6b897a6-9485-11e7-9278-9d0f246edcf5.JPG)

**Comment by Kenny2github** [comment]
This seems to be caused by the height of the Scratch News box - on narrower screens the box is taller and the "What's Happening?" box has the same height as it.

Possible fix would be allow more than five items to be shown in that box...?

**Comment by DeleteThisAcount** [comment]
I know this may sound a little silly but does anyone know where the code is for the what's happening box?

**Comment by VutonDesign** [comment]
> I know this may sound a little silly but does anyone know where the code is for the what's happening box?

That's what I was wondering. 😆

**Comment by TheGrits** [comment]
> I know this may sound a little silly but does anyone know where the code is for the what's happening box?
> > That's what I was wondering. 😆
My gut tells me it's in the scratchr2 repository, which isn't accessible to the public.

**Comment by VutonDesign** [comment]
@TheGrits Ah

**Comment by towerofnix** [comment]
@TheGrits More likely in scratch-api (also a private repository), since the "what's happening?" box has been migrated to use the new API (`api.scratch.mit.edu` instead of `scratch.mit.edu/site-api`).

**Comment by chrisgarrity** [comment]
Actually the limit is in the getActivity function: https://github.com/LLK/scratch-www/blob/e8671c1727fd5af2e8ee55182cb51b03074f4c24/src/views/splash/splash.jsx#L74

**Comment by VutonDesign** [comment]
@chrisgarrity Perfect, If we were to change that to six I think the bottom user's profile image would be cut off slightly. If after testing it this does happen, We could make the 6 profile images a bit smaller to fit them all in.

**Comment by GreenBayRules** [comment]
I don't think we should do that, because 5 is like a common number... Idk how to explain it but i think you understand what im saying. Usually multiples of 5 are used, lol I can't word this properly

**Comment by Kenny2github** [comment]
I'd say make that number dynamic depending on the actual height of the box... but I don't know how to do that.

**Comment by kyleplo** [comment]
Unrelated, but why is the nav orange?

**Comment by Kenny2github** [comment]
@kyleplo that's the header for the "staging" server, a private version of the website that uses the `develop` branch rather than `master`.

**Comment by kyleplo** [comment]
@Kenny2github That's what I though

**Comment by joker314** [comment]
I would _request_ six from the API, but display either 5 or 6 depending on height. (Possibly add a "View More" to see more activity? but that's an aside...)
